### ISSUE TITLE
Segfault on start (Wayland)

### ISSUE THREAD

**Issue by imjasonmiller** [obsproject/obs-studio#4569]
### Operating System Info

Other

### Other OS

Arch Linux x86_64 (5.11.15-arch1-2)

### OBS Studio Version

27.0.0-rc2

### OBS Studio Version (Other)

OBS Studio - 27.0.0.rc2.r29.g00fdec249 (linux)

### OBS Studio Log URL

https://gist.github.com/imjasonmiller/8cbcb92bdc49a6f38134ff6f1bca8a25

### Expected Behavior

It should start.

### Current Behavior

The application crashes immediately or briefly (< 100 ms) shows a broken user interface and then crashes using either of the following commands:

```none
QT_QPA_PLATFORM=wayland obs
```

```none
QT_QPA_PLATFORM=wayland-egl obs
```

### Output 

![Screenshot of Open Broadcaster Software crashing and showing a broken interface](https://user-images.githubusercontent.com/12190184/115596517-7d138900-a2d8-11eb-9985-1c13883d2482.png)

<details>
  <summary>Display textual output in above screenshot</summary>
  
```none
Attempted path: share/obs/obs-studio/locale/en-US.ini
Attempted path: /usr/share/obs/obs-studio/locale/en-US.ini
Attempted path: share/obs/obs-studio/themes/Dark.qss
Attempted path: /usr/share/obs/obs-studio/themes/Dark.qss
info: Command Line Arguments: --verbose
info: Platform: Wayland
info: CPU Name: AMD Ryzen 5 2600 Six-Core Processor
info: CPU Speed: 2800.000MHz
info: Physical Cores: 6, Logical Cores: 12
info: Physical Memory: 16000MB Total, 6815MB Free
info: Kernel Version: Linux 5.11.15-arch1-2
info: Distribution: "Arch Linux" Unknown
info: Session Type: wayland
info: Portable mode: false
Attempted path: share/obs/obs-studio/themes/Dark/no_sources.svg
Attempted path: /usr/share/obs/obs-studio/themes/Dark/no_sources.svg
QMetaObject::connectSlotsByName: No matching signal for on_tbar_position_valueChanged(int)
QMetaObject::connectSlotsByName: No matching signal for on_actionShowTransitionProperties_triggered()
QMetaObject::connectSlotsByName: No matching signal for on_actionHideTransitionProperties_triggered()
info: OBS 27.0.0.rc2.r29.g00fdec249 (linux)
info: ---------------------------------
info: ---------------------------------
info: audio settings reset:
	samples per sec: 48000
	speakers:        2
info: ---------------------------------
info: Initializing OpenGL...
info: Using EGL/Wayland
info: Initialized EGL 1.5
info: Loading up OpenGL on adapter AMD AMD Radeon (TM) RX 480 Graphics (POLARIS10, DRM 3.40.0, 5.11.15-arch1-2, LLVM 11.1.0)
info: OpenGL loaded successfully, version 4.6 (Core Profile) Mesa 21.0.2, shading language 4.60
info: ---------------------------------
info: video settings reset:
	base resolution:   1920x1080
	output resolution: 1280x720
	downscale filter:  Bicubic
	fps:               30/1
	format:            NV12
	YUV mode:          709/Partial
info: NV12 texture support not available
info: Audio monitoring device:
	name: Default
	id: default
info: ---------------------------------
warning: Failed to load 'en-US' text for module: 'decklink-captions.so'
warning: Failed to load 'en-US' text for module: 'decklink-ouput-ui.so'
libDeckLinkAPI.so: cannot open shared object file: No such file or directory
warning: A DeckLink iterator could not be created.  The DeckLink drivers may not be installed
info: No blackmagic support
warning: v4l2loopback not installed, virtual camera disabled
info: [obs-browser]: Version 2.14.1
info: [obs-browser]: CEF Version 88.2.8+ge484012+chromium-88.0.4324.150
error: os_dlopen(libnvidia-encode.so.1->libnvidia-encode.so.1): libnvidia-encode.so.1: cannot open shared object file: No such file or directory

info: FFMPEG VAAPI supported
info: VLC found, VLC video source enabled
info: ---------------------------------
info:   Loaded Modules:
info:     vlc-video.so
info:     text-freetype2.so
info:     sndio.so
info:     rtmp-services.so
info:     obs-x264.so
info:     obs-vst.so
info:     obs-transitions.so
info:     obs-outputs.so
info:     obs-libfdk.so
info:     obs-filters.so
info:     obs-ffmpeg.so
info:     obs-browser.so
info:     linux-v4l2.so
info:     linux-pulseaudio.so
info:     linux-jack.so
info:     linux-decklink.so
info:     linux-capture.so
info:     linux-alsa.so
info:     image-source.so
info:     frontend-tools.so
info:     decklink-ouput-ui.so
info:     decklink-captions.so
info: ---------------------------------
info: ==== Startup complete ===============================================
info: All scene data cleared
info: ------------------------------------------------
info: pulse-input: Server name: 'pulseaudio 14.2'
info: pulse-input: Audio format: s16le, 44100 Hz, 2 channels
info: pulse-input: Started recording from 'alsa_output.pci-0000_0a_00.3.analog-stereo.monitor'
info: [Loaded global audio device]: 'Desktop Audio'
info: Switched to scene 'Scene'
info: ------------------------------------------------
info: Loaded scenes:
info: - scene 'Scene':
info: ------------------------------------------------
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
Attempted path: share/obs/obs-studio/images/overflow.png
Attempted path: /usr/share/obs/obs-studio/images/overflow.png
[1]    144051 segmentation fault (core dumped)
```

</details>

<details>
  <summary>Display journalctl output</summary>

```none
journalctl --user --priority=2 --reverse --since=2021-04-21 
```
```none
systemd-coredump[150286]: Process 150247 (obs) of user 1000 dumped core.
                                                
stack trace of thread 150278:
#0  0x00007f410ffb20af gs_create_texture_file_data (libobs.so.0 + 0x2b0af)
#1  0x00007f410ffc2f2f gs_texture_create_from_file (libobs.so.0 + 0x3bf2f)
#2  0x000056119edd3120 n/a (obs + 0x1c7120)
#3  0x000056119ecc9141 n/a (obs + 0xbd141)
#4  0x00007f411002bd5a render_display (libobs.so.0 + 0xa4d5a)
#5  0x00007f4110036569 obs_graphics_thread_loop (libobs.so.0 + 0xaf569)
#6  0x00007f4110037559 obs_graphics_thread (libobs.so.0 + 0xb0559)
#7  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#8  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

stack trace of thread 150247:
#0  0x00007f410f6b6e75 n/a (libqt5core.so.5 + 0x355e75)
#1  0x00007f410f6b953e n/a (libqt5core.so.5 + 0x35853e)
#2  0x00007f410f6bc21f _zn16qxmlstreamreader8readnextev (libqt5core.so.5 + 0x35b21f)
#3  0x00007f41121853dc _zn11qsvghandler5parseev (libqt5svg.so.5 + 0x303dc)
#4  0x00007f4112192b6f _zn16qsvgtinydocument4loaderk7qstring (libqt5svg.so.5 + 0x3db6f)
#5  0x00007f41121939ac _zn12qsvgrenderer4loaderk7qstring (libqt5svg.so.5 + 0x3e9ac)
#6  0x00007f4106e2f6c2 n/a (libqsvgicon.so + 0x36c2)
#7  0x00007f4106e30982 n/a (libqsvgicon.so + 0x4982)
#8  0x00007f4106e2ff04 n/a (libqsvgicon.so + 0x3f04)
#9  0x00007f410fa81d30 _znk5qicon10actualsizeep7qwindowrk5qsizens_4modeens_5statee (libqt5gui.so.5 + 0x1c9d30)
#10 0x00007f41102e7501 n/a (libqt5widgets.so.5 + 0x278501)
#11 0x00007f41102e79ab _znk17qdockwidgetlayout11titleheightev (libqt5widgets.so.5 + 0x2789ab)
#12 0x00007f41102e95e7 _znk17qdockwidgetlayout15sizefromcontenterk5qsizeb (libqt5widgets.so.5 + 0x27a5e7)
#13 0x00007f41102e9953 n/a (libqt5widgets.so.5 + 0x27a953)
#14 0x00007f41102ed637 n/a (libqt5widgets.so.5 + 0x27e637)
#15 0x00007f41102ed443 n/a (libqt5widgets.so.5 + 0x27e443)
#16 0x00007f41102f31a6 n/a (libqt5widgets.so.5 + 0x2841a6)
#17 0x00007f41102f476e n/a (libqt5widgets.so.5 + 0x28576e)
#18 0x00007f4110329b5b n/a (libqt5widgets.so.5 + 0x2bab5b)
#19 0x00007f41101e80e8 _zn14qlayoutprivate8doresizeev (libqt5widgets.so.5 + 0x1790e8)
#20 0x00007f41101e965e _zn7qlayout8activateev (libqt5widgets.so.5 + 0x17a65e)
#21 0x00007f41101c9745 _zn19qapplicationprivate13notify_helperep7qobjectp6qevent (libqt5widgets.so.5 + 0x15a745)
#22 0x00007f410f61768a _zn16qcoreapplication15notifyinternal2ep7qobjectp6qevent (libqt5core.so.5 + 0x2b668a)
#23 0x00007f410f61a183 _zn23qcoreapplicationprivate16sendpostedeventsep7qobjectip11qthreaddata (libqt5core.so.5 + 0x2b9183)
#24 0x00007f410f670b54 n/a (libqt5core.so.5 + 0x30fb54)
#25 0x00007f410ac1b02c g_main_context_dispatch (libglib-2.0.so.0 + 0x5402c)
#26 0x00007f410ac6eb59 n/a (libglib-2.0.so.0 + 0xa7b59)
#27 0x00007f410ac18781 g_main_context_iteration (libglib-2.0.so.0 + 0x51781)
#28 0x00007f410f670191 _zn20qeventdispatcherglib13processeventse6qflagsin10qeventloop17processeventsflagee (libqt5core.so.5 + 0x30f191)
#29 0x00007f410f615fec _zn10qeventloop4exece6qflagsins_17processeventsflagee (libqt5core.so.5 + 0x2b4fec)
#30 0x00007f410f61e4a4 _zn16qcoreapplication4execev (libqt5core.so.5 + 0x2bd4a4)
#31 0x000056119ec8960f main (obs + 0x7d60f)
#32 0x00007f410ee5cb25 __libc_start_main (libc.so.6 + 0x27b25)
#33 0x000056119ec8ab0e n/a (obs + 0x7eb0e)

stack trace of thread 150248:
#0  0x00007f410ef2937f __poll (libc.so.6 + 0xf437f)
#1  0x00007f410ac6eae8 n/a (libglib-2.0.so.0 + 0xa7ae8)
#2  0x00007f410ac18781 g_main_context_iteration (libglib-2.0.so.0 + 0x51781)
#3  0x00007f410ac187d2 n/a (libglib-2.0.so.0 + 0x517d2)
#4  0x00007f410ac490c1 n/a (libglib-2.0.so.0 + 0x820c1)
#5  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#6  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

stack trace of thread 150252:
#0  0x00007f410f0179ba __futex_abstimed_wait_common64 (libpthread.so.0 + 0x159ba)
#1  0x00007f410f011574 pthread_cond_timedwait@@glibc_2.3.2 (libpthread.so.0 + 0xf574)
#2  0x00007f410f4351c8 _zn14qwaitcondition4waitep6qmutex14qdeadlinetimer (libqt5core.so.5 + 0xd41c8)
#3  0x00007f410f432674 n/a (libqt5core.so.5 + 0xd1674)
#4  0x00007f410f42f07f n/a (libqt5core.so.5 + 0xce07f)
#5  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#6  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

stack trace of thread 150249:
#0  0x00007f410ef2937f __poll (libc.so.6 + 0xf437f)
#1  0x00007f410ac6eae8 n/a (libglib-2.0.so.0 + 0xa7ae8)
#2  0x00007f410ac1a593 g_main_loop_run (libglib-2.0.so.0 + 0x53593)
#3  0x00007f410b746558 n/a (libgio-2.0.so.0 + 0x102558)
#4  0x00007f410ac490c1 n/a (libglib-2.0.so.0 + 0x820c1)
#5  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#6  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

stack trace of thread 150255:
#0  0x00007f410f0179ba __futex_abstimed_wait_common64 (libpthread.so.0 + 0x159ba)
#1  0x00007f410f011574 pthread_cond_timedwait@@glibc_2.3.2 (libpthread.so.0 + 0xf574)
#2  0x00007f410ffe6631 os_event_timedwait (libobs.so.0 + 0x5f631)
#3  0x00007f411001d1b5 obs_hotkey_thread (libobs.so.0 + 0x961b5)
#4  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#5  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

stack trace of thread 150254:
#0  0x00007f410f0179ba __futex_abstimed_wait_common64 (libpthread.so.0 + 0x159ba)
#1  0x00007f410f011574 pthread_cond_timedwait@@glibc_2.3.2 (libpthread.so.0 + 0xf574)
#2  0x00007f410f4351c8 _zn14qwaitcondition4waitep6qmutex14qdeadlinetimer (libqt5core.so.5 + 0xd41c8)
#3  0x00007f410f432674 n/a (libqt5core.so.5 + 0xd1674)
#4  0x00007f410f42f07f n/a (libqt5core.so.5 + 0xce07f)
#5  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#6  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)
                                                
Stack trace of thread 150251:
#0  0x00007f410f0179ba __futex_abstimed_wait_common64 (libpthread.so.0 + 0x159ba)
#1  0x00007f410f011574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
#2  0x00007f410f4351c8 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd41c8)
#3  0x00007f410f432674 n/a (libQt5Core.so.5 + 0xd1674)
#4  0x00007f410f42f07f n/a (libQt5Core.so.5 + 0xce07f)
#5  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#6  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

Stack trace of thread 150256:
#0  0x00007f410ef2937f __poll (libc.so.6 + 0xf437f)
#1  0x00007f410ac6eae8 n/a (libglib-2.0.so.0 + 0xa7ae8)
#2  0x00007f410ac18781 g_main_context_iteration (libglib-2.0.so.0 + 0x51781)
#3  0x00007f410f670191 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x30f191)
#4  0x00007f410f615fec _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2b4fec)
#5  0x00007f410f42de92 _ZN7QThread4execEv (libQt5Core.so.5 + 0xcce92)
#6  0x00007f4107ea4098 n/a (libQt5DBus.so.5 + 0x17098)
#7  0x00007f410f42f07f n/a (libQt5Core.so.5 + 0xce07f)
#8  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#9  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)

Stack trace of thread 150253:
#0  0x00007f410f0179ba __futex_abstimed_wait_common64 (libpthread.so.0 + 0x159ba)
#1  0x00007f410f011574 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf574)
#2  0x00007f410f4351c8 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xd41c8)
#3  0x00007f410f432674 n/a (libQt5Core.so.5 + 0xd1674)
#4  0x00007f410f42f07f n/a (libQt5Core.so.5 + 0xce07f)
#5  0x00007f410f00b299 start_thread (libpthread.so.0 + 0x9299)
#6  0x00007f410ef34053 __clone (libc.so.6 + 0xff053)
```

The output goes on for thousands of lines in a similar fashion.

</details>

### Install

Get `PKGBUILD`

```none
cd ~/.cache/yay/
yay --getpkgbuild obs-studio-git
cd obs-studio-git
```

Modify `PKGBUILD` by adding `DENABLE_PIPEWIRE` as per [this comment](https://aur.archlinux.org/packages/obs-studio-git/#comment-800893).

```diff
- -DOBS_VERSION_OVERRIDE=$pkgver ..
+ -DOBS_VERSION_OVERRIDE=$pkgver /
+ -DENABLE_PIPEWIRE=ON ..
```

Build package

```none
makepkg --install --syncdeps
```

```
(1/1) installing obs-studio-git [###################################] 100%
Optional dependencies for obs-studio-git
    libfdk-aac: FDK AAC codec support [installed]
    libxcomposite: XComposite capture support [installed]
    libva-intel-driver: hardware encoding
    libva-mesa-driver: hardware encoding
    vlc: VLC Media Source [installed]
    luajit: Lua scripting [installed]
    python: Python scripting [installed]
    v4l2loopback-dkms: Virtual webcam [installed]
    pipewire: Pipewire capture [installed]
    xdg-desktop-portal: Pipewire capture [installed]
```

### Anything else we should know?

I'm assuming I forgot something along the way while installing or my guess would be is that there's something wrong with my `Qt` packages, but I'm out of my depth here…

If there's a need for more information, I'd love to hear so.

#### Dependencies

* [`sway-git`](https://aur.archlinux.org/packages/sway-git/) (r6671.ecfd6879-1)
* [`wlroots-git`](https://aur.archlinux.org/packages/wlroots-git/) (0.13.0.r26.g3c6826df-1)
* [`ffmpeg-git`](https://aur.archlinux.org/packages/ffmpeg-git/) (N-102003-g718e03e5f2)
* [`qt5-svg`](https://archlinux.org/packages/extra/x86_64/qt5-svg/) (5.15.2+kde+r7-1)
* [`qt5-x11extras`](https://archlinux.org/packages/extra/x86_64/qt5-x11extras/) (5.15.2-1)
* [`qt5-wayland-bugfixport-git`](https://aur.archlinux.org/packages/qt5-wayland-bugfixport-git/) (5.15.2[e3879c17]-2)
* [`pipewire`](https://archlinux.org/packages/extra/x86_64/pipewire/) (0.3.25-1)
* [`x264`](https://archlinux.org/packages/extra/x86_64/x264/) (3:0.161.r3039.544c61f-1)

I can list more, if needed.

Some disparities between my log file and the one of [this issue](https://github.com/obsproject/obs-studio/issues/4554) seems to be that I'm missing the following line:

```
09:09:25 AM.530: encoder 'simple_h264_recording' (obs_x264) created
```

Running `ls /etc/pipewire/media-session.d/` also gives me the following, with no files that have [`.pacnew` extensions](https://github.com/obsproject/obs-studio/issues/4554#issuecomment-822455022):

```
alsa-monitor.conf
bluez-monitor.conf
media-session.conf
v4l2-monitor.conf
```

---

Thanks a ton for all the effort and Georges in particular for the Wayland support! I'm looking forward to getting it to work.

**Comment by kkartaltepe** [comment]
Looks like a bug in Qt. Please capture the backtrace with gdb for sane symbols.

```
Process 150247 (obs) of user 1000 dumped core.

stack trace of thread 150247:
#0  0x00007f410f6b6e75 n/a (libqt5core.so.5 + 0x355e75)
#1  0x00007f410f6b953e n/a (libqt5core.so.5 + 0x35853e)
#2  0x00007f410f6bc21f _zn16qxmlstreamreader8readnextev (libqt5core.so.5 + 0x35b21f)
#3  0x00007f41121853dc _zn11qsvghandler5parseev (libqt5svg.so.5 + 0x303dc)
#4  0x00007f4112192b6f _zn16qsvgtinydocument4loaderk7qstring (libqt5svg.so.5 + 0x3db6f)
#5  0x00007f41121939ac _zn12qsvgrenderer4loaderk7qstring (libqt5svg.so.5 + 0x3e9ac)
#6  0x00007f4106e2f6c2 n/a (libqsvgicon.so + 0x36c2)
#7  0x00007f4106e30982 n/a (libqsvgicon.so + 0x4982)
#8  0x00007f4106e2ff04 n/a (libqsvgicon.so + 0x3f04)
#9  0x00007f410fa81d30 _znk5qicon10actualsizeep7qwindowrk5qsizens_4modeens_5statee (libqt5gui.so.5 + 0x1c9d30)
#10 0x00007f41102e7501 n/a (libqt5widgets.so.5 + 0x278501)
#11 0x00007f41102e79ab _znk17qdockwidgetlayout11titleheightev (libqt5widgets.so.5 + 0x2789ab)
#12 0x00007f41102e95e7 _znk17qdockwidgetlayout15sizefromcontenterk5qsizeb (libqt5widgets.so.5 + 0x27a5e7)
#13 0x00007f41102e9953 n/a (libqt5widgets.so.5 + 0x27a953)
#14 0x00007f41102ed637 n/a (libqt5widgets.so.5 + 0x27e637)
#15 0x00007f41102ed443 n/a (libqt5widgets.so.5 + 0x27e443)
#16 0x00007f41102f31a6 n/a (libqt5widgets.so.5 + 0x2841a6)
#17 0x00007f41102f476e n/a (libqt5widgets.so.5 + 0x28576e)
#18 0x00007f4110329b5b n/a (libqt5widgets.so.5 + 0x2bab5b)
#19 0x00007f41101e80e8 _zn14qlayoutprivate8doresizeev (libqt5widgets.so.5 + 0x1790e8)
#20 0x00007f41101e965e _zn7qlayout8activateev (libqt5widgets.so.5 + 0x17a65e)
#21 0x00007f41101c9745 _zn19qapplicationprivate13notify_helperep7qobjectp6qevent (libqt5widgets.so.5 + 0x15a745)
#22 0x00007f410f61768a _zn16qcoreapplication15notifyinternal2ep7qobjectp6qevent (libqt5core.so.5 + 0x2b668a)
#23 0x00007f410f61a183 _zn23qcoreapplicationprivate16sendpostedeventsep7qobjectip11qthreaddata (libqt5core.so.5 + 0x2b9183)
#24 0x00007f410f670b54 n/a (libqt5core.so.5 + 0x30fb54)
#25 0x00007f410ac1b02c g_main_context_dispatch (libglib-2.0.so.0 + 0x5402c)
#26 0x00007f410ac6eb59 n/a (libglib-2.0.so.0 + 0xa7b59)
#27 0x00007f410ac18781 g_main_context_iteration (libglib-2.0.so.0 + 0x51781)
#28 0x00007f410f670191 _zn20qeventdispatcherglib13processeventse6qflagsin10qeventloop17processeventsflagee (libqt5core.so.5 + 0x30f191)
#29 0x00007f410f615fec _zn10qeventloop4exece6qflagsins_17processeventsflagee (libqt5core.so.5 + 0x2b4fec)
#30 0x00007f410f61e4a4 _zn16qcoreapplication4execev (libqt5core.so.5 + 0x2bd4a4)
#31 0x000056119ec8960f main (obs + 0x7d60f)
#32 0x00007f410ee5cb25 __libc_start_main (libc.so.6 + 0x27b25)
#33 0x000056119ec8ab0e n/a (obs + 0x7eb0e)
```

**Comment by imjasonmiller** [comment]
Hi @kkartaltepe, here's an update in the meantime, as I'm new at this.

I edited the `PKGBUILD` to enable debug symbols:

```diff
+ options=("debug" "!strip")
```

```diff
-    -DENABLE_PIPEWIRE=ON ..
+    -DENABLE_PIPEWIRE=ON \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
```

Then ran it once again, followed by `coredumpctl debug`  with `gdb` being the default debugger, which gave me:

```none
warning: Unable to find libthread_db matching inferior's thread library, thread debugging will not be available.
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib/libthread_db.so.1".
Core was generated by `obs'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  ffmpeg_image_open_decoder_context (info=0x7ff2de7fb4e0) at /home/jason/.cache/yay/obs-studio-git/src/obs-studio-git/libobs/graphics/graphics-ffmpeg.c:33
33		AVCodec *const decoder = avcodec_find_decoder(codecpar->codec_id);
[Current thread is 1 (Thread 0x7ff2de7fc640 (LWP 222161))]
```

I'm somewhat surprised at it involving `ffmpeg` and not `Qt` on my system? Perhaps I did something wrong? If you can make sense of what might be happening I'd appreciate pointers. I'll continue looking into it tomorrow.

Thank you for your time!

**Comment by kkartaltepe** [comment]
Looks like your system packages are broken.

**Comment by imjasonmiller** [comment]
@kkartaltepe that as well! A closign update: I recompiled `ffmpeg` and that allowed it to start, but the interface ended up being broken:

![Broken Qt interface of Open Broadcaster Software](https://user-images.githubusercontent.com/12190184/115735509-d4723180-a38a-11eb-90df-4fc17fafa950.png)

I came across [this post](https://bbs.archlinux.org/viewtopic.php?pid=1968839#p1968839) by "Michal4k" who solved it by unsetting the `QT_WAYLAND_FORCE_DPI` variable, which worked:

![Working Qt interface of Open Broadcaster Software with Pipewire showing the active monitor being recorded](https://user-images.githubusercontent.com/12190184/115736103-4fd3e300-a38b-11eb-9f5f-d87396d90f09.png)

Thanks for the pointers and this can be considered a resolved.

Have a great week.
